kestra-io/kestra
# Feature/scope filter rename

### What changes are being made and why?

**Changes made:**

- Added a rename option to the `ScopeFilterButton` component in `ScopeFilterButton.vue`.
- Integrated the `ScopeFilterButton` component into parent components:
  - `Flows.vue`
  - `FlowMetrics.vue`
  - `FlowRoot.vue`
- Updated localization files to include new strings for the rename functionality.

**Why:**

To provide users with the ability to rename scope filters directly from the UI, enhancing the user experience and flexibility.

Relates to #6238

### How have the changes been QAed?

**QA Steps:**

1. Load the pages that contain the `Flows.vue`, `FlowMetrics.vue`, and `FlowRoot.vue` components.
2. Verify that the `ScopeFilterButton` component is displayed with the rename option.
3. Test the rename functionality by renaming a scope filter and ensuring the new name is displayed correctly.
4. Ensure that the components and the pages still function as expected without any errors.

### Setup Instructions

No additional setup required.
